sentry-ruby/lib/sentry/event.rb

```diff
@@ -71,8 +69,6 @@ def initialize(configuration:, integration_meta: nil, message: nil)
       @rack_env_whitelist = configuration.rack_env_whitelist
 
       @message = (message || "").byteslice(0..MAX_MESSAGE_SIZE_IN_BYTES)
```

Comment:
Ideally, `message` should be moved to `ErrorEvent` as well. But since that'll require changing `Event#initialize` as well and doesn't provide much benefit. I decided not to go this far.